src/blog/2024/04/building-an-admin-panel-in-node-red-with-dashboard-2.md

```diff
@@ -0,0 +1,111 @@
+---
+title: How to Build an Admin Dashboard with Node-RED Dashboard 2.0
+subtitle: A guide to building an Admin Panel in Node-RED with Dashboard 2.0
+description: Discover step-by-step instructions for developing an admin-only page in Node-RED Dashboard 2.0 using the FlowFuse Multiuser addon. Additionally, learn how to secure Dashboard 2.0 and explore the features of the FlowFuse multiuser addon.
+date: 2024-04-05
+authors: ["sumit-shinde"]
+image: 
+tags:
+   - posts
+   - dashboard 2.0
+
+---
+
+As businesses grow, data doubles, and managing and analyzing such increasing amounts of data becomes crucial. businesses are using Node-RED and its Dashboard 2.0 to handle this effectively. But what about controlling who can access what data? That's where an admin-only page comes in. Now With Node-RED Dashboard 2.0, we can also create robust and secure admin-only pages easily. In this guide, we'll provide you with step-by-step instructions to Build an Admin-only page with Node-RED Dashboard 2.0.
+
+<!--more-->
+
+If you're new to Dashboard 2.0, refer to our blog post [Getting Started with Dashboard 2.0](https://flowfuse.com/blog/2024/03/dashboard-getting-started/), and Dashboard 2.0 to get things started.
+
+## Enabling FlowFuse User Authentication
```

Comment:
@joepavitt @sumitshinde-84 I think we should start to capture this in the main FF docs. What do you think?

Reply:
It would be perfect, but I am looking for documents. I am not able to find step-by-step instructions for it. Could you please direct me to where I can find them? I guess it should be here: https://flowfuse.com/docs/user/instance-settings/#security.

Reply:
Docs is even better indeed, feel free to update that doc.